### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by zhyiyu** [Konfekt/FastFold#70]
Hi, I installed FastFold using [vim-plug](https://github.com/junegunn/vim-plug):

<img width="407" alt="install" src="https://user-images.githubusercontent.com/26550166/177073170-edf6bea9-c358-4028-9dd9-69aa8ff29302.png">

I also setup `~/.vim/vimrc` file accordingly:

```vim
nnoremap <space> za

nmap zuz <Plug>(FastFoldUpdate)
let g:fastfold_savehook = 1
let g:fastfold_fold_command_suffixes =  ['x','X','a','A','o','O','c','C']
let g:fastfold_fold_movement_commands = [']z', '[z', 'zj', 'zk']

let g:markdown_folding = 1
let g:rst_fold_enabled = 1
let g:tex_fold_enabled = 1
let g:vimsyn_folding = 'af'
let g:xml_syntax_folding = 1
let g:javaScript_fold = 1
let g:sh_fold_enabled= 7
let g:zsh_fold_enable = 1
let g:ruby_fold = 1
let g:perl_fold = 1
let g:perl_fold_blocks = 1
let g:r_syntax_folding = 1
let g:rust_fold = 1
let g:php_folding = 1
let g:fortran_fold=1
let g:clojure_fold = 1
let g:baan_fold=1
autocmd FileType python setlocal foldmethod=indent
```

But the foldings in LaTeX files are still not recognized:

<img width="466" alt="fold" src="https://user-images.githubusercontent.com/26550166/177073487-0f35ae62-df43-47d1-b450-ca6b16be7a8c.png">

Is there anything that I did wrong?
Thanks in advance!

I also have [vimtex](https://github.com/lervag/vimtex) installed.
Please let me know if you need more information.

**Comment by Konfekt** [comment]
Hello, 

If you use the folds of vimtex, then Vim's built-in folds

```vim
let g:tex_fold_enabled = 1
```

and Vimtex's folds

```vim
g:vimtex_fold_enabled = 1
```

are mutually exclusive.
Maybe ensuring that exactly one of those options is `0` can solve the problem?
Other than that, ` :help vimtex-folding` might help analyzing it.

**Comment by zhyiyu** [comment]
Hi, thanks for your response. I may have not stated the issue clearly. Let me try again.

I found that the `vimtex` foldings can be very slow for large files, therefore I turned to `FastFold` for help, hoping that it could be faster. But I found installing `FastFold` makes no difference. Did I install/set it incorrectly or `FastFold` can not be used together with `vimtex`?

Thanks again, please let me know if you need more information.

**Comment by Konfekt** [comment]
I am using vimtex with FastFold and it works. What output does hitting `zuz`produce?

**Comment by zhyiyu** [comment]
It produces:

```vim
'manual' folds already continuously updated
```

**Comment by Konfekt** [comment]
Then neither the built-in nor Vimtex's folding method was enabled. 

The option
```
let g:tex_fold_enabled = 1
```
should have enabled syntax folding, therefore
```vim
  setlocal foldmethod=syntax
```
should make folds work in your TeX file.

If not, or if you prefer to use Vimtex's folding, add
```vim
g:vimtex_fold_enabled = 1
```
to add autocommands that run
```vim
  set foldmethod=expr
  set foldexpr=vimtex#fold#level(v:lnum)
  set foldtext=vimtex#fold#text()
```
in every Tex file to enable Vimtex's folding.

**Comment by zhyiyu** [comment]
Thanks, I think I misunderstood what `FastFold` does, I thought it provides folding recognitions. While it actually restricts when foldings are updated, is that right?
